.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+install:
+  - npm config set //registry.npmjs.org/:_authToken=$NPM_TOKEN -q
+  - npm install --verbose
+script:
+  - npm test
```

Comment:
That requires that you have a login for snyk setup nowadays... so we'd need another key in travis ... but yeah I think it might be worth it 

![yes](https://img.shields.io/badge/willfix-✓-green.svg?style=flat-square)